src/components/Button/Button.module.css

```diff
@@ -121,6 +121,106 @@
   }
 }
 
+/**
```

Comment:
The secondary and tertiary button styles are just copies of the primary one for now as placeholders. Will address in a follow-up PR.